arango/collection.py

```diff
@@ -1590,6 +1590,7 @@ def update_many(
         sync: Optional[bool] = None,
         silent: bool = False,
         refill_index_caches: Optional[bool] = None,
+        raise_on_document_error: bool = False,
```

Comment:
Not sure whether to use `Optional[bool] = None` or `bool = False` here, given the mix of default values for some of the parameters in this function. Could use a second opinion

Reply:
Since one of the two will happen a `None` doesn't really have any meaning? A boolean type makes sense here imo